kratos/mpi/tests/test_mpi_model_part.py

```diff
@@ -0,0 +1,122 @@
+import KratosMultiphysics
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.mpi as KratosMPI
+import KratosMultiphysics.MetisApplication as KratosMetis # TODO refactor this by using the "distributed_import_model_part_utility"

```

Comment:
I have solved this quite good [here](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/MappingApplication/tests/mapper_test_case.py)

Reply:
:+1: Changin it. To be honest I copy pasted it from another test :P

Reply:
I know - I wrote this comment :D

In the long run I was thinking of a special `TestCase` that provides this method